mod-orders-storage/examples/purchase_order_get.sample

```diff
@@ -3,6 +3,10 @@
   "adjustment": "9b3be694-6716-4e14-b81d-8d76f0ae4146",
   "approved": true,
   "assigned_to": "ab18897b-0e40-4f31-896b-9c9adc979a88",
+  "close_reason": {
+    "reason" : "CLOSED WITH NO REASON"
```

Comment:
Just curious if you're using ALL CAPS for a reason, or just by convention?  If this was a requirement from Dennis/A-M, we should enforce it via a pattern in the schema...

Reply:
@craigmcnally There are no restrictions for reason field at this time, it's just an upper case sample. Sorry for the confusion. I will better change samples according to "sentence case" before merging